frontend/src/components/ui/SellMenu.jsx

```diff
@@ -0,0 +1,83 @@
+import * as THREE from 'three';
+import { useState, useEffect } from "react";
+import { Cone, Text } from '@react-three/drei';
+
+import { convertToRenderCoordinates } from "../../utils/render_utils";
+import { TOWERS } from "./BuyMenu";
+import { BUILD } from "../../Game";
+import { Status as TowerStatus } from "../../entities/towers/Tower";
+
+const NAME = "SellMenu";
+
+const RADIUS = 0.2;
+const HEIGHT = 0.4;
+const RADIAL_SEGMENTS = 6;
+
+export default function SellMenu({ game }) {
+    const { gameMap, mouseInput } = game;
+    const { width, depth } = gameMap;
+    const [selectedTower, setSelectedTower] = useState(null);
+
+    const reset = () => {
```

Comment:
Not a dealbreaker, but it's a pretty generic function name and doesn't clean up much vs just calling setSelectedTower()

Reply:
I was anticipating more in there but luckily things turned out simpler. I'll just get rid of it.